eclair-core/src/test/scala/fr/acinq/eclair/router/RouterSpec.scala

```diff
@@ -130,10 +130,10 @@ class RouterSpec extends BaseRouterSpec {
   test("handle bad signature for NodeAnnouncement") { fixture =>
     import fixture._
     val sender = TestProbe()
-    val buggy_ann_a = ann_a.copy(signature = ann_b.signature, timestamp = ann_a.timestamp + 1)
-    sender.send(router, PeerRoutingMessage(null, remoteNodeId, buggy_ann_a))
-    sender.expectMsg(TransportHandler.ReadAck(buggy_ann_a))
-    sender.expectMsg(InvalidSignature(buggy_ann_a))
+    val buggy_ann_b = ann_b.copy(signature = ann_c.signature, timestamp = ann_b.timestamp + 1)
```

Comment:
Why did you need to change that?

Reply:
Good catch that was some leftover from development, in 3aa79f7 i reverted the changes to RouterSpec and BaseRouterSpec (along with updating the copyright header).